pkg/commands/config.go

```diff
@@ -75,7 +74,7 @@ func ListConfigs(ioStreams cmdutil.IOStreams, cmd *cobra.Command) error {
 	if err != nil {
 		return err
 	}
-	table := uitable.New()
+	table := newUITable()
 	table.MaxColWidth = 60
```

Comment:
some is 60, while you defined 80. Can you try to verify all of these commands whether they should use 60 or 80. I prefer 80, but I'm afraid some of them defined to 60 because there're too many columns.

Reply:
got it, I also prefer 80 and will check all.